requirements.txt

```diff
@@ -26,8 +26,7 @@ thredds_crawler>=1.5.4
 pyyaml>=5.2
 rq==1.3.0
 rq-dashboard==0.3.4
-# pin IOOS QC to a particular commit on main until stable version with fixes is released
--e git+https://github.com/ioos/ioos_qc@57813de111af201f8d7b68c7fcd6337bf37dd6ed#egg=ioos_qc
+git+https://github.com/ioos/ioos_qc.git@v2.2.0#egg=ioos_qc
```

Comment:
Can we use the stable version from PyPI or conda instead of the GitHub release? GH releases are, sadly, not immutable and if by mistake someone edits that, it may cause some really hard to debug breakages.